src/main/java/cms/gov/madie/measure/services/TestCaseService.java

```diff
@@ -457,6 +458,9 @@ public CopyTestCaseResult copyTestCasesToMeasure(
     for (TestCase sourceTestCase : sourceTestCases) {
       TestCase dupTestCase = sourceTestCase.deepCopy();
 
+      // Check and update any fields with proper timestamp
+      dupTestCase.setJson(convertDateTimeToUTC(dupTestCase.getJson()));
```

Comment:
I think the method `copyTestCasesToMeasure` is for both QDM and QICore. but this fix should be applied to only QICore test cases

Reply:
Added an if check for model